specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanager.json

```diff
@@ -684,6 +684,187 @@
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "HeatMap-GET": {
+            "$ref": "./examples/HeatMap-GET.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/trafficManagerRealUserMetricsKey": {
+      "get": {
+        "tags": [
+          "RealUserMetrics"
+        ],
+        "operationId": "RealUserMetricsKey_Get",
+        "description": "Get the subscription-level Real User Metrics key for the given subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The subscription-level Traffic Manager Real User Metrics key.",
+            "schema": {
+              "$ref": "#/definitions/RealUserMetricsKeyModel" 
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "RealUserMetricsKey-GET": {
+            "$ref": "./examples/RealUserMetricsKey-GET.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "RealUserMetrics"
+        ],
+        "operationId": "RealUserMetricsKey_CreateOrUpdate",
+        "description": "Create or update a Traffic Manager Real User Metrics key.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [""]
+            },
+            "description": "This body should be empty, Traffic Manager will provide a new key in the returned response."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The created or updated Traffic Manager Real User Metrics key.",
+            "schema": {
+              "$ref": "#/definitions/RealUserMetricsKeyModel" 
+            }
+          },
+          "201": {
+            "description": "The created or updated Traffic Manager Real User Metrics key.",
+            "schema": {
+              "$ref": "#/definitions/RealUserMetricsKeyModel" 
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
```

Comment:
Is this a long running operation? if yes mark it so using ` "x-ms-long-running-operation": true`